pkg/sdk/views_gen.go

```diff
@@ -84,10 +88,47 @@ type AlterViewOptions struct {
 	SetTagsOnColumn               *ViewSetColumnTags             `ddl:"keyword"`
 	UnsetTagsOnColumn             *ViewUnsetColumnTags           `ddl:"keyword"`
 }
+type DoubleQuotedString struct {
```

Comment:
hmmm, I don't like this abstraction, we shouldn't have content information inside the type

Reply:
What do you recommend, then? This field is case sensitive.

Reply:
why not a normal string?

Reply:
When we use unquoted strings, Snowflake makes them uppercase (similar to IDs), so we wouldn't be able to support all names here.

Reply:
But what does the name have to do with the logic?

Reply:
We discussed this and the struct will be renamed to Column (or similarly) in the next PR. I will keep this thread open till then.